homeassistant/components/tibber/sensor.py

```diff
@@ -416,6 +417,25 @@ async def async_update(self) -> None:
         self._attr_extra_state_attributes["price_level"] = price_level
         self._attr_extra_state_attributes["intraday_price_ranking"] = price_rank
 
+        class PriceItem(TypedDict):
+            time: datetime
+            price: float
+            level: str
+
+        levels = self._tibber_home.price_level
+        prices: list[PriceItem] = [
+            {"time": datetime.fromisoformat(k), "price": price, "level": levels[k]}
+            for (k, price) in self._tibber_home.price_total.items()
+        ]
+        # Prices come from a dict which is not guaranteed to be ordered.
+        prices.sort(key=lambda e: e["time"])
+
+        today = datetime.now().date()
+        prices_today = [p for p in prices if p["time"].date() == today]
+        prices_tomorrow = [p for p in prices if p["time"].date() > today]
+        self._attr_extra_state_attributes["prices_today"] = prices_today
+        self._attr_extra_state_attributes["prices_tomorrow"] = prices_tomorrow
```

Comment:
We try to avoid state attributes in general and we don't allow setting state that isn't the current state.